docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -1389,6 +1389,20 @@ supported as a space-separated list. For example,
     variant: "A B C D"
 ```
 
+[Task/variant tags](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#task-and-variant-tags) 
```

Comment:
Since this is in the same file, you can link this by just using `#task-and-variant-tags` rather than spelling out the full URL.